lib/common_test/test/ct_netconfc_SUITE_data/netconfc_remote_SUITE.erl

```diff
@@ -77,9 +77,7 @@ end_per_suite(Config) ->
 %% Running the netconf server in a remote node, test that the client
 %% process terminates if the remote node goes down.
 remote_crash(Config) ->
-    {ok,Node} = ct_slave:start(nc_remote_crash,[{boot_timeout,15},
-						{init_timeout,15},
-						{startup_timeout,15}]),
+    {ok, _Controller, Node} = ?CT_PEER(#{name => nc_remote_crash, timeout => timer:seconds(15)}),
```

Comment:
Where is the option `timeout` coming from to ?CT_PEER? I only see wait_boot here: https://www.erlang.org/docs/28/apps/stdlib/peer.html#t:start_options/0